NitroxClient/MonoBehaviours/Gui/MainMenu/MainMenuMultiplayerPanel.cs

```diff
@@ -81,15 +81,19 @@ private void CreateServerButton(string text, string joinIp, string joinPort)
                 OpenJoinServerMenu(joinIp, joinPort);
             });
 
-            GameObject delete = Instantiate(deleteButtonRef, multiplayerButtonInst.transform, false);
-            Button deleteButtonButton = delete.GetComponent<Button>();
-            deleteButtonButton.onClick = new Button.ButtonClickedEvent();
-            deleteButtonButton.onClick.AddListener(() =>
+            // Probably temporary solution but we don't want LAN servers to be deleted
+            if (!isLan)
```

Comment:
I prefer changing this name so indicate which purpose it has as opposed to what it is.
```suggestion
            if (!isReadOnly)
```